zerver/lib/user_groups.py

```diff
@@ -81,10 +85,16 @@ def check_remove_user_from_user_group(user_profile: UserProfile, user_group: Use
 
 
 def create_user_group(
-    name: str, members: List[UserProfile], realm: Realm, description: str = ""
+    name: str,
+    members: List[UserProfile],
+    realm: Realm,
+    description: str = "",
+    is_system_group: bool = False,
```

Comment:
Can we put `*` before `description` to require these last parameters to be keyword-only?